create_v1alpha2_machine.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
Maybe we should have `v1alpha1` and `v1alpha2` directories for these scripts, to keep them more clearly separated?  I know you put `v1alpha2` in the name of another one, but what do you think about directories instead?

Reply:
Good idea! I will add new scripts directory and directories for both v1alpha1 and v1alpha2